### DISCUSSION TITLE
Semaphore .tar.gz file for macOS M1 macbook

### DISCUSSION THREAD

**Discussion by timmy2pillows** [semaphoreui/semaphore#1058] [Q&A]
Hi everyone.  I'm new to semaphore and I'm trying to install it using the binary file method on my 2021 macbook pro with m1 max chip.  Which semaphore .tar.gz file do I need to download for my mac?  thanks!

**Comment by cf-sewe** [comment]
I dont have a Mac, but think you should be able to do this with semaphore in a container (Docker, Rancher...).
Semaphore is only available as linux arm64 build, I am pretty sure that will not work on MacOS directly.
